app/Main.hs

```diff
@@ -96,16 +96,18 @@ printDirectory (DirectoryPrefix dir) = do
     setSGR   [Reset]
 
 printTermResults :: Int -> (String, TermResults) -> IO ()
-printTermResults w (_, results) = do
+printTermResults w (_, results) =
     printMatches w results $ matches results
 
+likelihoodColor :: RemovalLikelihood -> Color
+likelihoodColor High = Red
```

Comment:
I'm curious what people prefer in cases like this where a guard would also be super simple and possibly less typing.